packages/preferences/src/browser/workspace-preference-provider.ts

```diff
@@ -69,7 +69,6 @@ export class WorkspacePreferenceProvider extends PreferenceProvider {
                     delegate.onDidPreferencesChanged(changes => this.onDidPreferencesChangedEmitter.fire(changes))
                 ]);
             }
```

Comment:
I've tested with different editor.fontSize in user and workspace settings and switching between no workspace and workspace, everything seems to be properly picked up without this event. 👍